### PR TITLE
Add ConoHa Provider

### PR DESCRIPTION
**Pull request by kaz**
Add a provider for [ConoHa DNS API](https://www.conoha.jp/docs/)

### CODE REVIEW

**Comment by CapsuleCD** [comment]
Hi.

I'm an automated pull request bot named [CapsuleCD](https://www.capsulecd.com). I handle testing, versioning and package releases for this project. 
- If you're the owner of this repo, you can click the button below to kick off the CapsuleCD build pipeline and create an automated release.'
- If not, don't worry, someone will be by shortly to check on your pull request. 

[CapsuleCD](https://www.capsulecd.com/project/github/AnalogJ/lexicon/pullrequests/306)

---
If you're interested in learning more about [CapsuleCD](http://www.github.com/AnalogJ/capsulecd), or adding it to your project, you can check it out [here](https://www.capsulecd.com)

File: lexicon/providers/conoha.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+import json
+import logging
+import time
+
+import requests
+
+from .base import Provider as BaseProvider
```
Comment (AnalogJ) [inline lexicon/providers/conoha.py]:
```suggestion
from lexicon.providers.base import Provider as BaseProvider
```

**Comment by kaz** [comment]
I applied your review comments, and rebased to the current master.
Thanks!

**Comment by AnalogJ** [comment]
Sorry for the delay getting this PR merged. We're working on updating the configuration system and I want to get that merged before adding new providers.

**Pull request merged**